stdlib/public/core/CompilerProtocols.swift

```diff
@@ -708,3 +708,67 @@ public protocol _DestructorSafeContainer {
 
 @available(*, unavailable, renamed: "Boolean")
 public typealias BooleanType = Boolean
+
+// Deprecated by SE-0115.
+
+@available(*, deprecated, renamed: "ExpressibleByNilLiteral")
+public typealias NilLiteralConvertible
+  = ExpressibleByNilLiteral
+@available(*, deprecated, renamed: "_ExpressibleByBuiltinIntegerLiteral")
+public typealias _BuiltinIntegerLiteralConvertible
+  = _ExpressibleByBuiltinIntegerLiteral
```

Comment:
I wasn't sure who was and wasn't using them.  They will be gone in the revision that properly deprecates them.